modules/tinymce/src/core/main/ts/fmt/FormatUtils.ts

```diff
@@ -18,39 +18,73 @@ const isNode = (node: any): node is Node => !!(node).nodeType;
 const isElementNode = (node: Node): node is Element =>
   NodeType.isElement(node) && !Bookmarks.isBookmarkNode(node) && !isCaretNode(node) && !NodeType.isBogus(node);
 
-const isInlineBlock = (node: Node): boolean => {
-  return node && /^(IMG)$/.test(node.nodeName);
+// Control blocks are any elements e.g. (img, iframe, noneditable element) that can be selected individually
+// In TinyMCE, selected blocks are indicated with the data-mce-selected attribute
+const isControlBlockSelected = (dom: DOMUtils, node: Node): boolean => {
```

Comment:
I think this function would be better suited in `ControlSelection` as it's responsible for defining how something is selected like this... Additionally, if someone changes the logic later they are more likely to see it being in the same spot instead of being scattered in a completely separate section of the code.

Reply:
Actually, the more I read this, this is not about Control Elements, as a noneditable element is **not** a control selection (remember control elements are ones that can be resized). It appears to just be an entire element selection. So maybe just rename this by removing `Control` and update the comment?

Reply:
Ah yep. My use of terms was incorrect. I'll try to figure out a better name. Thanks

Reply:
I ended up renaming it to `isElementDirectlySelected` as I think that makes it a bit clearer